istio/assets/dashboards/istio_overview.json

```
@@ -1,5 +1,5 @@
 {
-    "title": "Istio Dashboard",
+    "title": "Istio Dashboard (v < 1.5)",
```

```suggestion
    "title": "Istio Dashboard (< v1.5)",
```

reword to match other dashboard? https://github.com/DataDog/integrations-core/blob/master/istio/assets/dashboards/istio_1_5_overview.json#L2